tests/components/application-connector/Makefile

```diff
@@ -2,24 +2,43 @@ NAMESPACE ?= test
 GOPATH ?= $(shell go env GOPATH)
 
 GATEWAY_TEST_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/gateway-test:$(DOCKER_TAG)"
+VALIDATOR_TEST_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/app-connectivity-validator-test:$(DOCKER_TAG)"
 MOCK_APP_IMAGE = "$(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/mock-app:$(DOCKER_TAG)"
 
-.PHONY: release test-os test-skr test image publish clean image-gateway-test publish-gateway-test test-gateway test-gateway-debug image-mock-app publish-mock-app
+.PHONY: release test image publish clean image-gateway-test publish-gateway-test test-gateway test-gateway-debug image-mock-app publish-mock-app
 
-test: test-gateway
-image: image-gateway-test
-clean: clean-gateway-test
+test: test-gateway test-validator
+image: image-gateway-test image-app-connectivity-validator-test
+clean: clean-gateway-test clean-app-conn-validator-test
```

Comment:
I think we should keep the names consistent. I'd be `clean-validator-test` to be consistent with the `clean-gateway-test`. The same with the rest of the commands and image names